CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+## 0.0.6
```

Comment:
I would suggest we use this format:
https://keepachangelog.com

Reply:
done